src/components/node-list/node-list-items.test.js

```diff
@@ -97,17 +97,28 @@ describe('node-list-selectors', () => {
 
     it('filters expected number of items', () => {
       expect(filteredTagItems.length).not.toBe(tags.length);
-      expect(filteredTagItems).toHaveLength(1);
+      expect(filteredTagItems).toHaveLength(2);
     });
 
     it('returns items of the correct format', () => {
       expect(filteredTagItems).toEqual(tagItems);
     });
 
-    it('returns items for each tag', () => {
-      filteredTagItems.forEach((tagItem, index) => {
-        expect(tagItem.name).toEqual(tags[index].name);
-        expect(tagItem.id).toEqual(tags[index].id);
+    it('returns the filtered items that contains the search value', () => {
+      filteredTagItems.forEach(tagItem => {
+        expect(tagItem.name).toContain(searchValue);
+        expect(tagItem.id).toContain(searchValue);
+      });
+    });
+
+    it('filtered tagItems names and ids should match that of original tag', () => {
```

Comment:
What's the need to test this, do we think there's a chance they could get changed?
Also I thought previously we were testing the output of a specific input search, that seems missing now?

Reply:
Here is a bit of history behind the changes in this test and further updates I am making after discussions with @bru5 on slack:
1. I had removed the test (`'returns items for each tag'`) - the test uses the `index` of each run to reference the items in the tags array for comparison with the relative `tagItem` of that run, which does not seem to be testing anything. On discussion with @bru5 , we decided that this is a bunk test which could be deleted, while further adding modifications to allow a more comprehensive set of tests for testing the filtered tagItems.
2. I have added more tests on top of existing `expect` statements under the `'filters expected number of items'` test to ensure the right tags ( both 'small' and 'medium') gets returned.
3. further tests to ensure the filtered item contains the search term.